File: lib/fancy.ex
defmodule Fancy do
  def my_add(x, y) do
    z = x + y
    1 + z
  end

  def concat_questionmark(text) do
    abc = text <> "?"
    Enum.join([abc, "?", "?"])
  end

  def trim_test(text) do
    x = String.trim(text)
    String.trim(x, "abc")
  end
end

File: fancy.js
  const my_add = (x, y) => {
    let z = x + y;
    return 1 + z;
  }


  const concat_questionmark = (text) => {
    let abc = ''.concat(text, "?");
    return ''.concat(abc, "?", "?");
  }


  const trim_test = (text) => {
    let x = text.trim();
    return x.replace(RegExp(`^${"abc"}+`), '').replace(RegExp(`${"abc"}+$`), '');
  }
